indexd/index/drivers/alchemy.py

```diff
@@ -1096,6 +1096,31 @@ def get(self, did, expand=True):
 
             return record.to_document_dict()
 
+    def get_with_nonstrict_prefix(self, did, expand=True):
```

Comment:
could you add 1 or 2 sentences at the end of [this section](https://github.com/uc-cdis/indexd#distributed-resolution-utilizing-prefixes-in-guids) to mention which endpoints resolve regardless of prefix usage, and that for other endpoints you need to get the real `did`

Reply:
added